_includes/code/howto/manage-data.read-all-objects.py

```diff
@@ -42,7 +42,7 @@
 # =========================================
 
 # START ReadAllTenants
-multi_collection = client.collections.get("MultiTenancyCollection")
+multi_collection = client.collections.get("WineReviewMT")
```

Comment:
Could we call this WineReviewMulti (or similar) to highlight the different name?

Reply:
Could do. I've called it MT here https://pypi.org/project/weaviate-demo-datasets so that's what I was using. If we do, I have to refactor it there as well.